joomla/joomla-cms
# [4.3][Guided Tours] Administrator console unresponsive when CSS selector is invalid

When someone creating a tour enters a selector that is not valid, the tour does not start, produces an error in the browser console, nothing happens but the tour keeps trying to start until the user logs out. The user receives no feedback, unless he/her is aware of the browser console.

### Summary of Changes

We already caught the error but the session storage was not cleared.
This PR clears the storage on error and produces an error message for the user.

### Testing Instructions

Create a step or edit a step in a tour. Enter an invalid target, for instance `#1_tada`. This selector is invalid because it starts with a number.
Start the tour. you should see the error message:

![image](https://user-images.githubusercontent.com/5964177/223905091-8244dd0d-e6ba-4a3f-97b7-60e2fa81a51b.png)

### Actual result BEFORE applying this Pull Request

The tour does not start, there is no user feedback and the user needs to logout to recover.

### Expected result AFTER applying this Pull Request

The tour does not start, the administrator console is unaffected, a message pops up explaining that the tour cannot run and the user can resume activities.

### Link to documentations
Please select:
